### DISCUSSION TITLE
Bug Report for build-in (sass/scss) extenstions

### DISCUSSION THREAD

**Discussion by cyclingbyte** [microsoft/vscode-discussions#1580] [Extension Development QnA]
Hello, I now searched a while for how to report an issue to the buildin scss extension. I found out, that the extension where the plugin comes from is archived: https://github.com/atom/language-sass. Now I don't know where to report the bug and I hope you can help me here.

The following code will be compiled by sass without errors, but the VS Code buildin extension shows me a syntax error:
![VS code } expected](https://github.com/user-attachments/assets/94cc7b66-f2af-4e10-bd2b-2f4dd87f8001)
It says `} expeced` when I hover over `0%`

```sass
// Mixin for slide animations
@mixin slideX($from, $to) {
  0% {
    transform: translateX($from);
  }
  100% {
    transform: translateX($to);
  }
}

// Animations using the mixins
@keyframes slide-left {
  @include slideX(0, -100%);
}

@keyframes slide-right {
  @include slideX(-100%, 0);
}

@keyframes slide-left-two {
  @include slideX(0, 100%);
}

@keyframes slide-right-two {
  @include slideX(100%, 0);
}
```

**Comment by alefragnani** [comment]
Hi @cyclingbyte ,

If you are having issues with some built in extension in VS Code, the best place to report it is in the VS Code repo https://github.com/microsoft/vscode/issues/new/choose.

You could also use the `Help: Report Issue...` command inside VS Code, choosing `Visual Studio Code` in the _For:_ field in the form, to identify that the issue is with VS Code itself. 

By the way, for feature requests about VS Code usage, please also use the VS Code repository instead, as this Discussions repo is focused for extension development for now.

Hope this helps

**Reply by cyclingbyte** [reply]
Thank you.
I created an issue at microsoft/vscode/
